setup.py

```diff
@@ -278,6 +280,36 @@ def build_skia(build_base):
     return build_dir
 
 
+def get_skia_using_pkgconfig():
+    """Runs `pkg-config --libs --cflags skia` and parses returned
+    flags using argparse.
+    """
+    _parser = argparse.ArgumentParser()
+    _parser.add_argument("-I", dest="include_dirs", action="append", default=[])
+    _parser.add_argument("-L", dest="library_dirs", action="append", default=[])
+    _parser.add_argument("-l", dest="libraries", action="append", default=[])
+
+    if BUILD_SKIA_FROM_SOURCE:
+        return _parser.parse_known_args([])[0]
```

Comment:
why calling this at all `if BUILD_SKIA_FROM_SOURCE:`? There's nothing to parse

Reply:
The below logic depends on the return type from this function(or else there should be if conditions to check the return type).

Reply:
I see, but why do you need to call the argparse parser in this case? can't you simply return early on with an empty list? It's always going to be empty in this case.

Reply:
> can't you simply return early on with an empty list? It's always going to be empty in this case.

this function should return a class or similar which should have the attributes `include_dirs`, `library_dirs` and `libraries`. I think it's simple to get that by passing an empty list to argparse's parser rather than defining one our own.

Reply:
oh sorry, I didn't realise. OK then